packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -29,6 +29,7 @@
 
   .main-container {
     flex-grow: 1;
+    margin: var(--terra-consumer-layout-margin, 0);
```

Comment:
I feel like technically this should be padding..... (space between container and its content)